scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -130,7 +129,9 @@ class ThemeAggregator {
     }
 
     let themesToAggregate = defaultTheme ? [defaultTheme] : [];
-    themesToAggregate = themesToAggregate.concat(scoped);
+    if (scoped.length) {
```

Comment:
I think we actually just want to check for the existence of scoped.  If scoped is [] concat will work fine.

```suggestion
    if (scoped) {
```